lib/create-plugin.js

```diff
@@ -65,9 +81,24 @@ function createPlugin(instrumentationApi) {
 
               resolverSegment.name = `resolve: ${formattedPath}`
 
-              if (pathArray.length > longestResolvedPath.depth) {
-                longestResolvedPath.depth = pathArray.length
-                longestResolvedPath.formatted = formattedPath
+              if (pathArray.length > deepestResolvedPath.depth) {
+                deepestResolvedPath.depth = pathArray.length
+                deepestResolvedPath.formatted = formattedPath
+              }
+              resolverSegment.addAttribute(FIELD_PATH_ATTR, formattedPath)
+              resolverSegment.addAttribute(FIELD_NAME_ATTR, info.fieldName)
+              resolverSegment.addAttribute(RETURN_TYPE_ATTR, info.returnType.toString())
+              resolverSegment.addAttribute(PARENT_TYPE_ATTR, info.parentType.toString())
+
+
+              for (const [key, value] of Object.entries(args)) {
```

Comment:
Do you think there's value is adding args as attribute without a 'value'?

Reply:
I considered that but ultimately decided to keep it simple and follow a similar model to request params. We can always look to add later if we find there's a need.